services/ext/api.go

```diff
@@ -961,6 +961,10 @@ func (api *PublicAPI) SetDisplayName(ctx context.Context, displayName string) er
 	return api.service.messenger.SetDisplayName(displayName)
 }
 
+func (api *PublicAPI) SetBio(ctx context.Context, bio string) error {
```

Comment:
Just in case: it already exposed in settings service. 

https://github.com/status-im/status-go/blob/82ae52b73d0646bd7a74b127d86c967fc8998e27/services/accounts/settings.go#L169

Reply:
Thank you @osmaczko, I added a deprecated comment for SetBio in setting service